### ISSUE TITLE
twitch.tv: ads

### ISSUE THREAD

**Issue by MSH-Yeet** [uBlockOrigin/uAssets#11188]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png)
- [X] I have no custom filter lists, or I verified that the issue still occurs without custom filter lists
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)
- [X] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.twitch.tv/`

### Category

ads

### Describe the issue

basically ads keeps poping on Twitch.tv only for some reason, I never even checked what's in ubo's settings until today when I purged the cache + update, yet I still see ads when I visit a streamer or during watching a stream, and it's annoying.
I did notice that it's the same ad that keeps appear everytime, maybe because of my region (Morocco) or? idk.

also I did answer truthfully to all the checkpoints, almost all xD

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>
![download](https://user-images.githubusercontent.com/97084625/148025565-35f86a3c-80df-429c-8d9a-4f3b25055b87.png)

### Configuration

<details>

```yaml
uBlock Origin: 1.40.4
Chromium: 96
filterset (summary): 
  network: 78236
  cosmetic: 40271
  scriptlet: 16398
  html: 0
listset (total-discarded, last updated): 
  default: 
    easylist: 62576-9, 13m
    easyprivacy: 26599-17, 13m
    plowe-0: 3682-817, 13m
    ublock-abuse: 76-0, 13m
    ublock-badware: 3648-65, 13m
    ublock-filters: 30035-140, 13m
    ublock-privacy: 185-3, 13m
    ublock-unbreak: 1710-39, 13m
    urlhaus-1: 7628-0, 13m
filterset (user): [empty]
trustedset: 
  added: [array of 14 redacted]
  removed: 
    about-scheme
    chrome-extension-scheme
    chrome-scheme
    edge-scheme
    moz-extension-scheme
    opera-scheme
    vivaldi-scheme
    wyciwyg-scheme
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  launchToReadiness: 1533 ms (selfie)
popupPanel: 
  blocked: 28
```
</details>

**Comment by uBlock-user** [comment]
Duplicate of #10639

**Comment by gorhill** [comment]
> To avoid burdening **VOLUNTEERS** with invalid or duplicate reports, disrespecting these steps **WILL** result in your account being blocked on this repo (temporarily, then permanently with repeated disregard for the template).
> ...
> - [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
> ...
> - [X] I did not answer truthfully to **all** the above checkpoints
